Settings: make Peripherals a Standard setting, not Advanced

MilhouseVH opened the pull request:
This change will avoid [this situation](http://forum.kodi.tv/showthread.php?tid=269814&pid=2395203#pid2395203) where new users have trouble configuring CEC which is hidden by default, as it's an "Expert" setting.

Ping @jjd-uk.

MilhouseVH committed:
Settings: make Peripherals a Standard setting, not Advanced

jjd-uk commented:
I don't see the need for this to be a Level 0 Basic setting, since the default Level on install is Level 1 Standard I would prefer any change to be to Level 1.

MilhouseVH commented:
I assumed the default level would have been Basic, but since it's Standard I've updated the PR accordingly - thanks! Also changed the title as the current level isn't Expert, it's Advanced. Such a small change, so many mistakes. :)
